src/lxc/confile.c

```diff
@@ -2967,6 +2967,54 @@ static int lxc_get_item_network(struct lxc_conf *c, char *retv, int inlen)
 	return fulllen;
 }
 
+static int lxc_get_idmaps(struct lxc_conf *c, char *retv, int inlen)
+{
+	struct lxc_list *it;
+	int len, listlen, ret;
+	int fulllen = 0;
```

Comment:
What do you mean? The `strprint()` function expects `len`, `fulllen` and a bunch of others to be defined. I don't find it the best way to do it either.